{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "393bc9e8-af34-4ab0-876d-c7642b0d98ac",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "a7080d02-efac-4256-9ebd-986643ecd3d2",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DO Disease ID</th>\n",
       "      <th>DO Disease Name</th>\n",
       "      <th>OMIM IDs</th>\n",
       "      <th>Common Organism Name</th>\n",
       "      <th>NCBI Taxon ID</th>\n",
       "      <th>Symbol</th>\n",
       "      <th>EntrezGene ID</th>\n",
       "      <th>Mouse MGI ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>DOID:0112248</td>\n",
       "      <td>17-beta hydroxysteroid dehydrogenase 3 deficiency</td>\n",
       "      <td>OMIM:264300</td>\n",
       "      <td>human</td>\n",
       "      <td>9606</td>\n",
       "      <td>HSD17B3</td>\n",
       "      <td>3293.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DOID:0112248</td>\n",
       "      <td>17-beta hydroxysteroid dehydrogenase 3 deficiency</td>\n",
       "      <td>OMIM:264300</td>\n",
       "      <td>mouse, laboratory</td>\n",
       "      <td>10090</td>\n",
       "      <td>Hsd17b3</td>\n",
       "      <td>15487.0</td>\n",
       "      <td>MGI:107177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>DOID:0111453</td>\n",
       "      <td>2-aminoadipic 2-oxoadipic aciduria</td>\n",
       "      <td>OMIM:204750</td>\n",
       "      <td>human</td>\n",
       "      <td>9606</td>\n",
       "      <td>DHTKD1</td>\n",
       "      <td>55526.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>DOID:0111453</td>\n",
       "      <td>2-aminoadipic 2-oxoadipic aciduria</td>\n",
       "      <td>OMIM:204750</td>\n",
       "      <td>mouse, laboratory</td>\n",
       "      <td>10090</td>\n",
       "      <td>Dhtkd1</td>\n",
       "      <td>209692.0</td>\n",
       "      <td>MGI:2445096</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>DOID:0050573</td>\n",
       "      <td>2-hydroxyglutaric aciduria</td>\n",
       "      <td>NaN</td>\n",
       "      <td>human</td>\n",
       "      <td>9606</td>\n",
       "      <td>L2HGDH</td>\n",
       "      <td>79944.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18631</th>\n",
       "      <td>DOID:905</td>\n",
       "      <td>Zellweger syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>human</td>\n",
       "      <td>9606</td>\n",
       "      <td>PHYH</td>\n",
       "      <td>5264.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18632</th>\n",
       "      <td>DOID:905</td>\n",
       "      <td>Zellweger syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mouse, laboratory</td>\n",
       "      <td>10090</td>\n",
       "      <td>Pex1</td>\n",
       "      <td>71382.0</td>\n",
       "      <td>MGI:1918632</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18633</th>\n",
       "      <td>DOID:905</td>\n",
       "      <td>Zellweger syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mouse, laboratory</td>\n",
       "      <td>10090</td>\n",
       "      <td>Pex11b</td>\n",
       "      <td>18632.0</td>\n",
       "      <td>MGI:1338882</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18634</th>\n",
       "      <td>DOID:0060478</td>\n",
       "      <td>Zika fever</td>\n",
       "      <td>NaN</td>\n",
       "      <td>human</td>\n",
       "      <td>9606</td>\n",
       "      <td>STAT2</td>\n",
       "      <td>6773.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18635</th>\n",
       "      <td>DOID:0050782</td>\n",
       "      <td>Zollinger-Ellison syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>human</td>\n",
       "      <td>9606</td>\n",
       "      <td>EGF</td>\n",
       "      <td>1950.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>18636 rows Ã— 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      DO Disease ID                                    DO Disease Name  \\\n",
       "0      DOID:0112248  17-beta hydroxysteroid dehydrogenase 3 deficiency   \n",
       "1      DOID:0112248  17-beta hydroxysteroid dehydrogenase 3 deficiency   \n",
       "2      DOID:0111453                 2-aminoadipic 2-oxoadipic aciduria   \n",
       "3      DOID:0111453                 2-aminoadipic 2-oxoadipic aciduria   \n",
       "4      DOID:0050573                         2-hydroxyglutaric aciduria   \n",
       "...             ...                                                ...   \n",
       "18631      DOID:905                                 Zellweger syndrome   \n",
       "18632      DOID:905                                 Zellweger syndrome   \n",
       "18633      DOID:905                                 Zellweger syndrome   \n",
       "18634  DOID:0060478                                         Zika fever   \n",
       "18635  DOID:0050782                         Zollinger-Ellison syndrome   \n",
       "\n",
       "          OMIM IDs Common Organism Name  NCBI Taxon ID   Symbol  \\\n",
       "0      OMIM:264300                human           9606  HSD17B3   \n",
       "1      OMIM:264300    mouse, laboratory          10090  Hsd17b3   \n",
       "2      OMIM:204750                human           9606   DHTKD1   \n",
       "3      OMIM:204750    mouse, laboratory          10090   Dhtkd1   \n",
       "4              NaN                human           9606   L2HGDH   \n",
       "...            ...                  ...            ...      ...   \n",
       "18631          NaN                human           9606     PHYH   \n",
       "18632          NaN    mouse, laboratory          10090     Pex1   \n",
       "18633          NaN    mouse, laboratory          10090   Pex11b   \n",
       "18634          NaN                human           9606    STAT2   \n",
       "18635          NaN                human           9606      EGF   \n",
       "\n",
       "       EntrezGene ID Mouse MGI ID  \n",
       "0             3293.0          NaN  \n",
       "1            15487.0   MGI:107177  \n",
       "2            55526.0          NaN  \n",
       "3           209692.0  MGI:2445096  \n",
       "4            79944.0          NaN  \n",
       "...              ...          ...  \n",
       "18631         5264.0          NaN  \n",
       "18632        71382.0  MGI:1918632  \n",
       "18633        18632.0  MGI:1338882  \n",
       "18634         6773.0          NaN  \n",
       "18635         1950.0          NaN  \n",
       "\n",
       "[18636 rows x 8 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_mouse_data = pd.read_csv(\"MGI_DO.tsv\", sep=\"\\t\")\n",
    "df_mouse_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "22b80345-40e0-4732-ba80-10780709a060",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DO Disease Name</th>\n",
       "      <th>OMIM IDs</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>17-beta hydroxysteroid dehydrogenase 3 deficiency</td>\n",
       "      <td>OMIM:264300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>17-beta hydroxysteroid dehydrogenase 3 deficiency</td>\n",
       "      <td>OMIM:264300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2-aminoadipic 2-oxoadipic aciduria</td>\n",
       "      <td>OMIM:204750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2-aminoadipic 2-oxoadipic aciduria</td>\n",
       "      <td>OMIM:204750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>2-hydroxyglutaric aciduria</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18631</th>\n",
       "      <td>Zellweger syndrome</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18632</th>\n",
       "      <td>Zellweger syndrome</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18633</th>\n",
       "      <td>Zellweger syndrome</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18634</th>\n",
       "      <td>Zika fever</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18635</th>\n",
       "      <td>Zollinger-Ellison syndrome</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>18636 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                         DO Disease Name     OMIM IDs\n",
       "0      17-beta hydroxysteroid dehydrogenase 3 deficiency  OMIM:264300\n",
       "1      17-beta hydroxysteroid dehydrogenase 3 deficiency  OMIM:264300\n",
       "2                     2-aminoadipic 2-oxoadipic aciduria  OMIM:204750\n",
       "3                     2-aminoadipic 2-oxoadipic aciduria  OMIM:204750\n",
       "4                             2-hydroxyglutaric aciduria          NaN\n",
       "...                                                  ...          ...\n",
       "18631                                 Zellweger syndrome          NaN\n",
       "18632                                 Zellweger syndrome          NaN\n",
       "18633                                 Zellweger syndrome          NaN\n",
       "18634                                         Zika fever          NaN\n",
       "18635                         Zollinger-Ellison syndrome          NaN\n",
       "\n",
       "[18636 rows x 2 columns]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_mouse_disease_omims = df_mouse_data[[\"DO Disease Name\", \"OMIM IDs\"]]\n",
    "df_mouse_disease_omims"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "bc49452f-45aa-478d-b36d-bb356c71947c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DO Disease Name</th>\n",
       "      <th>OMIM IDs</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>17-beta hydroxysteroid dehydrogenase 3 deficiency</td>\n",
       "      <td>OMIM:264300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2-aminoadipic 2-oxoadipic aciduria</td>\n",
       "      <td>OMIM:204750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>3MC syndrome 1</td>\n",
       "      <td>OMIM:257920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>3MC syndrome 2</td>\n",
       "      <td>OMIM:265050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>3MC syndrome 3</td>\n",
       "      <td>OMIM:248340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18622</th>\n",
       "      <td>Y-linked deafness 2</td>\n",
       "      <td>OMIM:400047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18623</th>\n",
       "      <td>Y-linked spermatogenic failure 2</td>\n",
       "      <td>OMIM:415000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18624</th>\n",
       "      <td>Yoon-Bellen neurodevelopmental syndrome</td>\n",
       "      <td>OMIM:619701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18625</th>\n",
       "      <td>Yunis-Varon syndrome</td>\n",
       "      <td>OMIM:216340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18627</th>\n",
       "      <td>Zaki syndrome</td>\n",
       "      <td>OMIM:619648</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4435 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                         DO Disease Name     OMIM IDs\n",
       "0      17-beta hydroxysteroid dehydrogenase 3 deficiency  OMIM:264300\n",
       "2                     2-aminoadipic 2-oxoadipic aciduria  OMIM:204750\n",
       "6                                         3MC syndrome 1  OMIM:257920\n",
       "7                                         3MC syndrome 2  OMIM:265050\n",
       "8                                         3MC syndrome 3  OMIM:248340\n",
       "...                                                  ...          ...\n",
       "18622                                Y-linked deafness 2  OMIM:400047\n",
       "18623                   Y-linked spermatogenic failure 2  OMIM:415000\n",
       "18624            Yoon-Bellen neurodevelopmental syndrome  OMIM:619701\n",
       "18625                               Yunis-Varon syndrome  OMIM:216340\n",
       "18627                                      Zaki syndrome  OMIM:619648\n",
       "\n",
       "[4435 rows x 2 columns]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_mouse_disease_omims = df_mouse_disease_omims.dropna()\n",
    "df_mouse_disease_omims = df_mouse_disease_omims.drop_duplicates()\n",
    "df_mouse_disease_omims"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "e5f0e807-0f8b-4236-af4c-545faf06e25a",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_mouse_disease_omims.to_csv(\"unique_mouse_disease_omims.tsv\", index=None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "b2c09670-3636-4cf9-9dcc-1834b333a095",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DiseaseNames</th>\n",
       "      <th>OMIMIDs</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>17-beta hydroxysteroid dehydrogenase 3 deficiency</td>\n",
       "      <td>OMIM:264300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2-aminoadipic 2-oxoadipic aciduria</td>\n",
       "      <td>OMIM:204750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>3MC syndrome 1</td>\n",
       "      <td>OMIM:257920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3MC syndrome 2</td>\n",
       "      <td>OMIM:265050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>3MC syndrome 3</td>\n",
       "      <td>OMIM:248340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4895</th>\n",
       "      <td>Y-linked deafness 2</td>\n",
       "      <td>OMIM:400047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4896</th>\n",
       "      <td>Y-linked spermatogenic failure 2</td>\n",
       "      <td>OMIM:415000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4897</th>\n",
       "      <td>Yoon-Bellen neurodevelopmental syndrome</td>\n",
       "      <td>OMIM:619701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4898</th>\n",
       "      <td>Yunis-Varon syndrome</td>\n",
       "      <td>OMIM:216340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4899</th>\n",
       "      <td>Zaki syndrome</td>\n",
       "      <td>OMIM:619648</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4900 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                           DiseaseNames      OMIMIDs\n",
       "0     17-beta hydroxysteroid dehydrogenase 3 deficiency  OMIM:264300\n",
       "1                    2-aminoadipic 2-oxoadipic aciduria  OMIM:204750\n",
       "2                                        3MC syndrome 1  OMIM:257920\n",
       "3                                        3MC syndrome 2  OMIM:265050\n",
       "4                                        3MC syndrome 3  OMIM:248340\n",
       "...                                                 ...          ...\n",
       "4895                                Y-linked deafness 2  OMIM:400047\n",
       "4896                   Y-linked spermatogenic failure 2  OMIM:415000\n",
       "4897            Yoon-Bellen neurodevelopmental syndrome  OMIM:619701\n",
       "4898                               Yunis-Varon syndrome  OMIM:216340\n",
       "4899                                      Zaki syndrome  OMIM:619648\n",
       "\n",
       "[4900 rows x 2 columns]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "disease_names = list()\n",
    "omim_ids = list()\n",
    "\n",
    "for i, row in df_mouse_disease_omims.iterrows():\n",
    "    r_values = row.values\n",
    "    d_name = r_values[0]\n",
    "    d_omim = r_values[1].split(\"|\")\n",
    "    if len(d_omim) > 1:\n",
    "        for omim in d_omim:\n",
    "            disease_names.append(d_name)\n",
    "            omim_ids.append(omim)\n",
    "    else:\n",
    "        disease_names.append(d_name)\n",
    "        omim_ids.append(d_omim[0])\n",
    "        \n",
    "df_disease_names_omim = pd.DataFrame(zip(disease_names, omim_ids), columns=[\"DiseaseNames\", \"OMIMIDs\"])\n",
    "df_disease_names_omim"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "29066b2f-76aa-42d3-9fab-332f49e8f615",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DiseaseNames</th>\n",
       "      <th>OMIMIDs</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>17-beta hydroxysteroid dehydrogenase 3 deficiency</td>\n",
       "      <td>OMIM:264300</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2-aminoadipic 2-oxoadipic aciduria</td>\n",
       "      <td>OMIM:204750</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>3MC syndrome 1</td>\n",
       "      <td>OMIM:257920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3MC syndrome 2</td>\n",
       "      <td>OMIM:265050</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>3MC syndrome 3</td>\n",
       "      <td>OMIM:248340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4895</th>\n",
       "      <td>Y-linked deafness 2</td>\n",
       "      <td>OMIM:400047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4896</th>\n",
       "      <td>Y-linked spermatogenic failure 2</td>\n",
       "      <td>OMIM:415000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4897</th>\n",
       "      <td>Yoon-Bellen neurodevelopmental syndrome</td>\n",
       "      <td>OMIM:619701</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4898</th>\n",
       "      <td>Yunis-Varon syndrome</td>\n",
       "      <td>OMIM:216340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4899</th>\n",
       "      <td>Zaki syndrome</td>\n",
       "      <td>OMIM:619648</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4900 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                           DiseaseNames      OMIMIDs\n",
       "0     17-beta hydroxysteroid dehydrogenase 3 deficiency  OMIM:264300\n",
       "1                    2-aminoadipic 2-oxoadipic aciduria  OMIM:204750\n",
       "2                                        3MC syndrome 1  OMIM:257920\n",
       "3                                        3MC syndrome 2  OMIM:265050\n",
       "4                                        3MC syndrome 3  OMIM:248340\n",
       "...                                                 ...          ...\n",
       "4895                                Y-linked deafness 2  OMIM:400047\n",
       "4896                   Y-linked spermatogenic failure 2  OMIM:415000\n",
       "4897            Yoon-Bellen neurodevelopmental syndrome  OMIM:619701\n",
       "4898                               Yunis-Varon syndrome  OMIM:216340\n",
       "4899                                      Zaki syndrome  OMIM:619648\n",
       "\n",
       "[4900 rows x 2 columns]"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_disease_names_omim = df_disease_names_omim.drop_duplicates()\n",
    "df_disease_names_omim"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "87f537e1-5452-4cbe-8b20-c27719c80e6e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[Clinical phenotype and genetic analysis of six Chinese patients affected with Acromicric dysplasia due to variants of FBN1 gene].\tTo retrospectively analyze the clinical and genetic characteristics of six patients with Acromicric dysplasia due to variants of the FBN1 gene.\tSix patients who had visited the Affiliated Hospital of Qingdao University between February 2018 and October 2020 were selected as the study subjects. Clinical data of the patients were collected. High-throughput sequencing was carried out. And candidate variants were verified by Sanger sequencing.\tAll of the six patients had presented with severe short stature (< 3s), brachydactyly, short and broad hands and feet. Other manifestations included joint stiffness, facial dysmorphism, delayed bone age, liver enlargement, coracoid femoral head, and lumbar lordosis. Genetic testing revealed that all had harbored heterozygous variants of the FBN1 gene. Patient 1 had harbored a c.5183C>T (p.A1728V) missense variant in exon 42, which had derived from his father (patient 2). Patient 3 had harbored a c.5284G>A (p.G1762S) missense variant in exon 43, which had derived from her mother (patient 4). Patient 5 had harbored a c.5156G>T (p.C1719F) missense variant in exon 42, which was de novo in origin. Patient 6 had harbored a c.5272G>T (p.D1758Y) missense variant in exon 43, which was also de novo in origin. The variants carried by patients 1, 3 and 6 were known to be pathogenic. Based on the guidelines from the American College of Medical Genetics and Genomics (ACMG), the FBN1: c.5156G>T was rated as a pathogenic variant (PS2+PM1+PM2_Supporting +PM5+PP3).\tAll of the six patients had severe short stature and a variety of other clinical manifestations, which may be attributed to the variants of the FBN1 gene.\n",
      "Dental characteristics of patients with four different types of skeletal dysplasias.\tSkeletal dysplasia (SD) comprises more than 450 separate disorders. We hypothesized that their dental features would be distinctive and investigated the tooth characteristics of four patients with different SDs.\tFour SD patients with molecularly confirmed diagnoses, Pt-1 acromicric dysplasia, Pt-2 hypophosphatasia and hypochondroplasia, Pt-3 cleidocranial dysplasia, and Pt-4 achondroplasia, were recruited. A tooth from each patient was evaluated for mineral density (micro-computerized tomography), surface roughness (surface profilometer), microhardness, mineral contents (energy-dispersive X-ray), and ultrastructure (scanning electron microscopy and histology), and compared with three tooth-type matched controls.\tPt-1 and Pt-3 had several unerupted teeth. Pt-2 had an intact-root-exfoliated tooth at 2Â years old. The lingual surfaces of the patients' teeth were significantly smoother, while their buccal surfaces were rougher, than controls, except for Pt-1's buccal surface. The patients' teeth exhibited deep grooves around the enamel prisms and rough intertubular dentin. Pt-3 demonstrated a flat dentinoenamel junction and Pt-2 had an enlarged pulp, barely detectable cementum layer, and ill-defined cemento-dentinal junction. Reduced microhardnesses in enamel, dentin, and both layers were observed in Pt-3, Pt-4, and Pt-1, respectively. Pt-1 showed reduced Ca/P ratio in dentin, while both enamel and dentin of Pt-2 and Pt-3 showed reduced Ca/P ratio.\tEach SD has distinctive dental characteristics with changes in surface roughness, ultrastructure, and mineral composition of dental hard tissues.\tIn this era of precision dentistry, identifying the specific potential dental problems for each patient with SD would help personalize dental management guidelines.\n",
      "[Clinical manifestations and genetic analysis of 4 patients with variants of FBN1 gene].\tTo explore the genetic basis for four patients suspected for Marfan syndrome (MFS).\tFour male patients with suspected MFS and their family members who were treated at West China Second Hospital of Sichuan University from September 12, 2019 to March 27, 2021 were selected as the study subjects. Peripheral venous blood samples were collected from the patients and their parents or other pedigree members for the extraction of genomic DNA. Whole exome sequencing was carried out, and candidate variants were validated by Sanger sequencing. The pathogenicity of the variants was determined based on the guidelines from the American College of Medical Genetics and Genomics (ACMG).\tGenetic testing revealed that all four patients have harbored variants of the FBN1 gene, including c.430_433del (p.His144fs) deletional variant in exon 5, c.493C>T (p.Arg165*) nonsense variant in exon 6, c.5304_5306del (p.Asp1768del) deletional variant in exon 44 and c.5165C>G (p.Ser1722Cys) missense variant in exon 42. According to the ACMG guidelines, the c.430_433del and c.493C>T were classified as pathogenic variants (PVS1+PM2_Supporting+PP4; PVS1+PS1+PS2+PM2_Supporting+PP4). c.5304_5306del and c.5165C>G were classified as likely pathogenic variants (PS2+PM2_Supporting+PM4+PP4; PS2_Moderate+PS1+PM1+PM2_Supporting).\tThe c.430_433del and c.5304_5306del variants of the FBN1 gene identified in this study were unreported previously. Above results have enriched the variation spectrum of the FBN1 gene and provided a basis for genetic counseling and prenatal diagnosis of patients with MFS and acromicric dysplasia.\n",
      "Acromicric dysplasia caused by a mutation of fibrillin 1 in a family: A case report.\tAcromicric dysplasia (AD) is a rare skeletal dysplasia. Its incidence is < 1/1000000, and only approximately 60 cases are reported worldwide. It is a disease characterized by severe short stature, short hands and feet, facial abnormalities, normal intelligence, and bone abnormalities. Unlike other skeletal dysplasia, AD has a mild clinical phenotype, mainly characterized by short stature. Extensive endocrine examination has not revealed a potential cause. The clinical effect of growth hormone therapy is still uncertain.\tWe report a clinical phenotype of AD associated with mutations in the fibrillin 1 (FBN1) (OMIM 102370) gene c.5183C>T (p. Ala1728Val) in three people from a Chinese family. A 4-year-old member of the family first visited the hospital because of slow growth and short stature for 2 years, but no abnormalities were found after a series of laboratory tests, echocardiography, pituitary magnetic resonance imaging, and ophthalmological examination. Recombinant human growth hormone (rhGH) was used to treat the patient for > 5 years. The efficacy of rhGH was apparent in the first year of treatment; the height increased from -3.64 standard deviation score (SDS) to -2.88 SDS, while the efficacy weakened from the second year. However, long-term follow-up is required to clarify the efficacy of rhGH.\tFBN1-related AD has genetic heterogeneity and/or clinical variability, which brings challenges to the evaluation of clinical treatment. rhGH is effective for treatment of AD, but long-term follow-up is needed to clarify the effect.\n",
      "Acromicric dysplasia due to a novel missense mutation in the fibrillin 1 gene in a three-generation family.\tShort stature is one of the most common reasons for consulting a paediatric endocrinologist. Targeted diagnosis of familial short stature can be challenging due to a broad spectrum of differential diagnoses.\tHere we report a novel mutation in the fibrillin 1 gene (FBN1) in six family members causing a mild phenotype of acromicric dysplasia. Additionally, we present the effects of growth hormone therapy in one of the affected children.\tAcromicric dysplasia is a very rare skeletal dysplasia with a prevalence of <1 of 1.000.000 with only about 60 cases being reported worldwide. It is characterized by short stature, acromelia, mild facial dysmorphy but normal intelligence. This study aims to exemplify the clinical and molecular features of FBN1-related acromicric dysplasia and illustrates its pleiotropy by presenting a new, mild phenotype.\n",
      "A Review of Three Chinese Cases of Acromicric/Geleophysic Dysplasia with FBN1 Mutations.\tThis study aims to explore the clinical features and molecular diagnosis of FBN1-related acromelic dysplasia in Chinese patients.\tThe clinical and genetic features of three FBN1-related acromicric dysplasia (AD)/geleophysic dysplasia (GD) Chinese patients from two families were reviewed, and comprehensive medical evaluations were performed. Targeted next-generation sequencing was used to detect genetic mutations associated with short statures, including FBN1. Sanger sequencing was used to determine the de novo mutation origin.\tPatient 1 presented with short stature, short and stubby hands and feet, mild facial dysmorphism, hepatomegaly, delayed bone age and beak-like femoral heads. Patient 2 and this patient's father merely presented with short stature, wide and short hands, and beak-like femoral heads. One novel mutation, c.5272G>T(p.D1758Y), and one known mutation, c.5183C>T(p.A1728V), were identified in these patients.\tThe clinical features varied among these patients. The variant c.5272G>T(p.D1758Y) is a novel mutation.\n",
      "Sleep-disordered breathing and its management in children with rare skeletal dysplasias.\tSleep-disordered breathing (SDB) is common in patients with skeletal dysplasias. The aim of our study was to analyze SDB and respiratory management in children with rare skeletal dysplasias. We performed a retrospective analysis of patients with spondyloepiphyseal dysplasia congenita (SEDC), metatropic dysplasia (MD), spondyloepimetaphyseal dysplasia (SEMD), acrodysostosis (ADO), geleophysic dysplasia (GD), acromicric dysplasia (AD), and spondylocostal dysplasia (SCD) between April 2014 and October 2020. Polygraphic data, clinical management, and patients' outcome were analyzed. Thirty-one patients were included (8 SEDC, 3 MD, 4 SEMD, 1 ADO, 4 GD, 3Â AD, and 8 SCD). Sixteen patients had obstructive sleep apnea (OSA): 11 patients (2 with SEDC, 1 with SEMD, 1 with ADO, 1 with GD, 2 with AD, and 4 with SCD) had mild OSA, 2 (1 SEMD and 1 GD) had moderate OSA, and 3 (1 SEDC, 1 MD, 1 SEMD) had severe OSA. Adenotonsillectomy was performed in one patient with SCD and mild OSA, and at a later age in two other patients with ADO and AD. The two patients with moderate OSA were treated with noninvasive ventilation (NIV) because of nocturnal hypoxemia. The three patients with severe OSA were treated with adenotonsillectomy (1 SEDC), adeno-turbinectomy and continuous positive airway pressure (CPAP; 1 MD), and with NIV (1 SEMD) because of nocturnal hypoventilation. OSA and/or alveolar hypoventilation is common in patients with skeletal dysplasias, underlining the importance of systematic screening for SDB. CPAP and NIV are effective treatments for OSA and nocturnal hypoventilation/hypoxemia.\n",
      "Geleophysic and acromicric dysplasias: natural history, genotype-phenotype correlations, and management guidelines from 38 cases.\tGeleophysic dysplasia (GD) and acromicric dysplasia (AD) are characterized by short stature, short extremities, and progressive joint limitation. In GD, cardiorespiratory involvement can result in poor prognosis. Dominant variants in the FBN1 and LTBP3 genes are responsible for AD or GD, whereas recessive variants in the ADAMTSL2 gene are responsible for GD only. The aim of this study was to define the natural history of these disorders and to establish genotype-phenotype correlations.\tThis monocentric retrospective study was conducted between January 2008 and December 2018 in a pediatric tertiary care center and included patients with AD or GD with identified variants (FBN1, LTBP3, or ADAMTSL2).\tTwenty-two patients with GD (12 ADAMTSL2, 8 FBN1, 2 LTBP3) and 16 patients with AD (15 FBN1, 1 LTBP3) were included. Early death occurred in eight GD and one AD. Among GD patients, 68% presented with heart valve disease and 25% developed upper airway obstruction. No AD patient developed life-threatening cardiorespiratory issues. A greater proportion of patients with either a FBN1 cysteine variant or ADAMTSL2 variantsÂ had a poor outcome.\tGD and AD are progressive multisystemic disorders with life-threatening complications associated with specific genotype. A careful multidisciplinary follow-up is needed.\n",
      "Acromicric dysplasia with stiff skin syndrome-like severe cutaneous presentation in an 8-year-old boy with a missense FBN1 mutation: Case report and literature review.\tAcromicric dysplasia is a rare heritable short-stature syndrome with joint stiffness and varying degrees of cutaneous hardness. Stiff skin syndrome is a rare connective tissue disorder characterized by diffusely thick and hard skin from the time of birth. Heterozygous point mutations in the FBN1 have been proposed as the predominant cause of both diseases.\tBy performing skin biopsy, X-ray imaging, electrocardiography, as well as whole-genome sequencing and Sanger sequencing, we diagnosed an 8-year-old Chinese boy as acromicric dysplasia with severe skin stiffness caused by a heterogeneous mutation in the FBN1.\tThe patient presented with skin tightness, wrist and ankle stiffness, short stature and limbs, several deformed joints in the extremities, cone-shaped epiphyses, and distinct facial features. He also had a patent foramen ovale and frequent respiratory infections. Skin biopsy showed thickened dermis and excessive collagen aggregation. Alcian blue staining indicated dermal mucopolysaccharide deposition. Mutation analysis revealed a heterozygous missense mutation, c.5243G>A (p.Cys1748Tyr), in exon 42 of the FBN1.\tThis is a report about acromicric dysplasia with stiff skin syndrome-like severe cutaneous presentation caused by a single hotspot mutation, further revealing the gene pleiotropy of FBN1.\n",
      "Genotype-phenotype correlation and expansion of orodental anomalies in LTBP3-related disorders.\tThe latent transforming growth factor-beta-binding protein 3 (LTBP3), encoding extracellular matrix proteins, plays a role in skeletal formation. Mutations in LTBP3 have been associated with various types of skeletal dysplasia. We aimed to characterize clinical and molecular features of more patients with mutations in the gene, which may help suggest genotype-phenotype correlation. The first two East Asian patients with short stature, heart defects, and orodental anomalies having LTBP3 mutations were identified. Whole exome and Sanger sequencing revealed that the one with a novel heterozygous missense (c.2017G>T, p.Gly673Cys) mutation in LTBP3 had clinical features consistent with acromicric dysplasia (ACMICD). The variant was located in the highly conserved EGF-like calcium-binding domain adjacent to the single reported LTBP3 variant associated with ACMICD. This finding supports that LTBP3 is a disease gene for ACMICD. Another patient with a novel homozygous splice site acceptor (c.1721-2A>G) mutation in LTBP3 was affected with dental anomalies and short stature (DASS). Previously undescribed orodental features included multiple unerupted teeth, high-arched palate, and microstomia found in our patient with ACMICD, and extensive dental infection, condensing osteitis, and deviated alveolar bone formation in our patient with DASS. Our results and comprehensive reviews suggest a genotype-phenotype correlation: biallelic loss-of-function mutations cause DASS, monoallelic missense gain-of-function mutations in the EGF-like domain cause ACMICD, and monoallelic missense gain-of-function mutations with more drastic effects on the protein functions cause geleophysic dysplasia (GPHYSD3). In summary, we expand the phenotypic and genotypic spectra of LTBP3-related disorders, support that LTBP3 is a disease gene for ACMICD, and propose the genotype-phenotype correlation of LTBP3 mutations.\n",
      "Fibrillin protein pleiotropy: Acromelic dysplasias.\tThe fibrillins are large extracellular matrix molecules that polymerize to form microfibrils. Fibrillin microfibrils are distinctive architectural elements that are both ubiquitous in the connective tissue space and also unique, displaying tissue-specific patterns. Mutations in the genes for fibrillin-1 (FBN1) result in multiple distinct pleiotropic disorders. Most of the more than 3000 mutations known today in FBN1 cause the Marfan syndrome. Marfan mutations can occur in any of the 56 domains that compose fibrillin-1. In contrast, rare mutations in FBN1 that are confined to only certain domains cause several different types of acromelic dysplasia. These genetic disorders demonstrate that specific domains of fibrillin-1 perform roles important to musculoskeletal growth. Many of the phenotypes of acromelic dysplasias are the opposite of those found in Marfan syndrome. Knowledge of the functions and structural organization of fibrillin molecules within microfibrils is required to understand how one protein and one gene can be the basis for multiple genetic disorders.\n",
      "A report of three families with FBN1-related acromelic dysplasias and review of literature for genotype-phenotype correlation in geleophysic dysplasia.\tAcromelic dysplasia is a heterogeneous group of rare skeletal dysplasias characterized by distal limb shortening. Weill-Marchesani syndrome (WMS), Geleophysic dysplasia (GD) and Acromicric dysplasia (AD) are clinically distinct entities within this group of disorders and are characterized by short stature, short hands, stiff joints, skin thickening, facial anomalies, normal intelligence and skeletal abnormalities. Mutations of the Fibrillin-1 (FBN1) gene have been reported to cause AD, GD and related phenotypes. We reported three families with acromelic short stature. FBN1 analysis showed that all affected individuals carry a heterozygous missense mutation c.5284GÂ >Â A (p.Gly1762Ser) in exon 42 of the FBN1 gene. This mutation was previously reported to be associated with GD. We reviewed the literature and compared the clinical features of the patients with FBN1 mutations to those with A Distintegrin And Metalloproteinase with Thrombospondin repeats-like 2 gene (ADAMTSL2) mutations. We found that tip-toeing gait, long flat philtrum and thin upper upper lip were more consistently found in GD patients with ADAMTSL2 mutations than in those with FBN1 mutations. The results have shed some light on the phenotype-genotype correlation in this group of skeletal disorders. A large scale study involving multidisciplinary collaboration would be needed to consolidate our findings.\n",
      "Three cases of Japanese acromicric/geleophysic dysplasia with FBN1 mutations: a comparison of clinical and radiological features.\tAcromicric dysplasia (AD) and geleophysic dysplasia (GD) are rare skeletal dysplasias characterized by short stature, acromelia, joint contracture, hepatomegaly, hoarseness and respiratory distress. Compared with GD, AD presents with milder clinical and radiological features. Radiological findings of AD and GD consist of shortened tubular bones of the hands and feet, and deformed capital femoral epiphyses. The genetic cause of AD and some cases of GD was shown to be mutations in the transforming growth factor (TGF) Î²-binding protein-like domain 5 of the fibrillin 1 gene (FBN1), which is also mutated in Marfan syndrome. In the present study, we report and compare the highly varied clinical and radiological features of three Japanese AD/GD children. Our patients, harboring FBN1 mutations p.Tyr1699Cys, p.Ser1750Arg, and p.Gly1762Ser, shared common clinical symptoms such as severe short stature, acromelia and hepatomegaly. Short tubular bones of hands and deformities of femur heads are common radiological features of our patients.\n",
      "Acromicric Dysplasia Caused by a Novel Heterozygous Mutation of FBN1 and Effects of Growth Hormone Treatment.\n",
      "Two Patients with Severe Short Stature due to a FBN1 Mutation (p.Ala1728Val) with a Mild Form of Acromicric Dysplasia.\tAcromicric dysplasia (AD) and geleophysic dysplasia 2 (GD2) belong to the category of acromelic dysplasia syndromes, consisting of severe short stature, short hands and feet and skin thickening. Both can result from missense mutations in the transforming growth factor beta 5 domain of the fibrillin-1 gene (FBN1).\tTwo patients (P1 age 10, and P2 age 7) from unrelated families presented to their endocrinologist with severe short stature (approx. -4 SDS). They were otherwise asymptomatic and only had mild facial dysmorphisms. Extensive endocrine work-up did not reveal an underlying etiology. Exome sequencing was performed in each family.\tExome sequencing identified the presence of the same heterozygous missense variant c.C5183T (p.Ala1728Val) in the FBN1 gene in both P1 and P2. This variant was previously reported in a patient with GD2 and associated cardiac valvulopathy and hepatomegaly. Detailed clinical re-examination, cardiac and skeletal imaging did not reveal any abnormalities in P1 or P2 other than mild hip dysplasia.\tThis report broadens the phenotypic spectrum of growth disorders associated with FBN1 mutations. Identical mutations give rise to a wide phenotypic spectrum, ranging from isolated short stature to a more classic picture of GD2 with cardiac involvement, distinct facial dysmorphisms and various skeletal anomalies.\n",
      "Mutations in LTBP3 cause acromicric dysplasia and geleophysic dysplasia.\tAcromelic dysplasias are a group of disorders characterised by short stature, brachydactyly, limited joint extension and thickened skin and comprises acromicric dysplasia (AD), geleophysic dysplasia (GD), Myhre syndrome and Weill-Marchesani syndrome. Mutations in several genes have been identified for these disorders (including latent transforming growth factor Î² (TGF-Î²)-binding protein-2 (LTBP2), ADAMTS10, ADAMSTS17 and fibrillin-1 (FBN1) for Weill-Marchesani syndrome, ADAMTSL2 for recessive GD and FBN1 for AD and dominant GD), encoding proteins involved in the microfibrillar network. However, not all cases have mutations in these genes.\tIndividuals negative for mutations in known acromelic dysplasia genes underwent whole exome sequencing.\tA heterozygous missense mutation (exon 14: c.2087C>G: p.Ser696Cys) in latent transforming growth factor Î² (TGF-Î²)-binding protein-3 (LTBP3) was identified in a dominant AD family. Two distinct de novo heterozygous LTPB3 mutations were also identified in two unrelated GD individuals who had died in early childhood from respiratory failure-a donor splice site mutation (exon 12 c.1846+5G>A) and a stop-loss mutation (exon 28: c.3912A>T: p.1304*Cysext*12).\tThe constellation of features in these AD and GD cases, including postnatal growth retardation of long bones and lung involvement, is reminiscent of the null ltbp3 mice phenotype. We conclude that LTBP3 is a novel component of the microfibrillar network involved in the acromelic dysplasia spectrum.\n",
      "Skeletal manifestations of Marfan syndrome associated to heterozygous R2726W FBN1 variant: sibling case report and literature review.\tFBN1 (15q21.1) encodes fibrillin-1, a large glycoprotein which is a major component of microfibrils that are widely distributed in structural elements of elastic and non-elastic tissues. FBN1 variants are responsible for the related connective tissue disorders, grouped under the generic term of type-1 fibrillinopathies, which include Marfan syndrome (MFS), MASS syndrome (Mitral valve prolapse, Aortic enlargement, Skin and Skeletal findings, Acromicric dysplasia, Familial ectopia lentis, Geleophysic dysplasia 2, Stiff skin syndrome, and dominant Weill-Marchesani syndrome.\tTwo siblings presented with isolated skeletal manifestations of MFS, including severe pectus excavatum, elongated face, scoliosis in one case, and absence of other clinical features according to Ghent criteria diagnosis, were screened for detection of variants in whole FBN1 gene (65 exons). Both individuals were heterozygous for the R2726W variant. This variant has been previously reported in association with some skeletal features of Marfan syndrome in the absence of both tall stature and non-skeletal features. These features are consistent with the presentation of the siblings reported here.\tThe presented cases confirm that the R2726W FBN1 variant is associated with skeletal features of MFS in the absence of cardiac or ocular findings. These findings confirm that FBN1 variants are associated with a broad phenotypic spectrum and the value of sequencing in atypical cases.\n",
      "Children with short-limbed short stature in pediatric endocrinological services in Japan.\tShort-limbed short stature is a heterogeneous condition that can result from many diseases such as bone disorder, metabolic disease, and multiple malformation syndrome. We conducted a questionnaire survey of council members of the Japanese Society of Pediatric Endocrinology and doctors of affiliated hospitals in 2010 to investigate short-limbed short stature. Among 91 hospitals, responses were obtained from 61 hospitals (67% response rate). This study also examined data of 193 short-limbed short stature patients, among whom FGFR3-related chondrodysplasia such as achondroplasia (n = 109; 56.5%) was found the most frequently. Second to achondroplasia, hypochondroplasia (n = 47; 24.4%) was the most frequently observed. Along with achondroplasia and hypochondroplasia, 31 patients with disorders of 13 other kinds and six undiagnosed patients were identified. Genetic testing for hypochondroplasia was conducted for only 27.7% of all hypochondroplasia patients, although hypochondroplasia is a heterogeneous condition with many causes, only one of which is FGFR3 mutation. We conducted a genetic analysis of 25 patients who had been clinically diagnosed as having \"hypochondroplasia\". In these patients, other diseases such as acromicric dysplasia, geleophysic dysplasia, and Aarskog-Scott syndrome were included in addition to FGFR3-related hypochondroplasia (n = 10). Clinical diagnosis of each disorder causing short-limbed short stature is difficult. Therefore, not only clinical diagnosis but also genetic diagnosis play an important role in the diagnosis of short-limb short stature. Diagnostic strategies must be created for each disorder.\n",
      "Three novel mutations of the FBN1 gene in Chinese children with acromelic dysplasia.\tGeleophysic dysplasia (GD), acromicric dysplasia (AD) and Weill-Marchesani syndrome (WMS) are rare disorders with overlapping characteristics, such as short stature, short hands and feet, joint limitations, skin thickening, mild facial anomalies, normal intelligence and abnormal skeletal symptoms, with GD distinct by progressive cardiac valvular thickening and WMS distinct by microspherophakia and ectopia lentis. Mutations in FBN1 gene have been identified in AD, GD and WMS patients. By targeted next-generation sequencing of skeletal dysplasia-related genes, including FBN1 and ADAMTSL2, three novel missense mutations, c.5189A>T (p.N1730I), c.5198G>T (p.C1733F), c.5243G>T (p.C1748F), and one known mutation c.5198G>A (p.C1733Y) of FBN1 gene were identified in four probands, respectively. Clinically, p.C1733Y was associated with GD, as reported previously, as well as the novel p.N1730I, whereas p.C1733F and p.C1748F were associated with AD and WMS. Interestingly, different mutations at the same codon (p.C1733Y and p.C1733F) were associated with different phenotypes (GD and AD, respectively). However, the mutations p.C1748F and p.C1748R were associated with WMS. Our data support the importance of TGFÎ²-binding protein-like domain 5 of FBN1 protein in pathogenicity of acromelic dysplasia, and expands the genotype/phenotype relations of these rare forms of fibrilliopathies.\n",
      "Leri's pleonosteosis, a congenital rheumatic disease, results from microduplication at 8q22.1 encompassing GDF6 and SDC2 and provides insight into systemic sclerosis pathogenesis.\tLeri's pleonosteosis (LP) is an autosomal dominant rheumatic condition characterised by flexion contractures of the interphalangeal joints, limited motion of multiple joints, and short broad metacarpals, metatarsals and phalanges. Scleroderma-like skin thickening can be seen in some individuals with LP. We undertook a study to characterise the phenotype of LP and identify its genetic basis.\tWhole-genome single-nucleotide polymorphism genotyping in two families with LP defined microduplications of chromosome 8q22.1 as the cause of this condition. Expression analysis of dermal fibroblasts from affected individuals showed overexpression of two genes, GDF6 and SDC2, within the duplicated region, leading to dysregulation of genes that encode proteins of the extracellular matrix and downstream players in the transforming growth factor (TGF)-Î² pathway. Western blot analysis revealed markedly decreased inhibitory SMAD6 levels in patients with LP. Furthermore, in a cohort of 330 systemic sclerosis cases, we show that the minor allele of a missense SDC2 variant, p.Ser71Thr, could confer protection against disease (p<1Ã—10(-5)).\tOur work identifies the genetic cause of LP in these two families, demonstrates the phenotypic range of the condition, implicates dysregulation of extracellular matrix homoeostasis genes in its pathogenesis, and highlights the link between TGF-Î²/SMAD signalling, growth/differentiation factor 6 and syndecan-2. We propose that LP is an additional member of the growing 'TGF-Î²-pathies' group of musculoskeletal disorders, which includes Myhre syndrome, acromicric dysplasia, geleophysic dysplasias, Weill-Marchesani syndromes and stiff skin syndrome. Identification of a systemic sclerosis-protective SDC2 variant lays the foundation for exploration of the role of syndecan-2 in systemic sclerosis in the future.\n",
      "Orthopedics management of acromicric dysplasia: follow up of nine patients.\tAcromicric dysplasia (AD) is an autosomal dominant disorder characterized by short stature, short extremities, stiff joint and skeleton features including brachymetacarpia, cone-shaped epiphyses, internal notch of the femoral head, and delayed bone age. Recently, we identified fibrillin 1 (FBN1) as the disease gene of AD. The aim of our study was to further describe the long-term follow up of AD patients with an emphasis on orthopedic management. Nine patients with FBN1 mutations were included in the study ranging in age from 5.5 to 64 years. For all, detailed clinical and radiological data were available.\tBirth parameters were always normal and patients progressively developed short stature <-3 SD. Carpal tunnel syndrome was observed in four patients. We found discrepancy between the carpal bone age and the radius and ulna epiphysis bone ages, a variable severity of hip dysplasia with acetabular dysplasia, epiphyseal and metaphyseal femoral dysplasia resembling Legg-Perthes-CalvÃ© disease and variable pelvic anteversion and hyperlordosis. Orthopedic surgery was required in two patients for hip dysplasia, in one for limb lengthening and in three for carpal tunnel syndrome. Our observations expand the AD phenotype and emphasize the importance of regular orthopedic survey.\n",
      "Similarity of geleophysic dysplasia and Weill-Marchesani syndrome.\tThe acromelic dysplasias comprise short stature, hands and feet, and stiff joints. Three disorders are ascribed to this group, namely Weill-Marchesani syndrome, geleophysic dysplasia, and acromicric dysplasia, although similar in phenotype, can be distinguished clinically. Weill-Marchesani syndrome, on the basis of microspherophakia and ectopia lentis; geleophysic dysplasia by progressive cardiac valvular thickening, tracheal stenosis, and/or bronchopulmonary insufficiency, often leading to early death. Microspherophakia has not been reported previously in geleophysic dysplasia. Mutations in FBN1, ADAMTS10, or ADAMTS17 cause Weill-Marchesani syndrome by disrupting the microfibrillar environment, while geleophysic dysplasia is associated with enhanced TGF-Î² signaling mediated through mutations in FBN1 or ADAMTSL2. We studied a 35-year-old woman with geleophysic dysplasia, with short stature, small hands and feet, limitation of joint mobility, mild skin thickening, cardiac valvular disease, restrictive pulmonary disease, and microspherophakia. Sequencing of ADAMTSL2 demonstrated two changes: IVS8-2A>G consistent with a disease-causing mutation, and IVS14-7G>A with potential to generate a new splice acceptor site and result in aberrant mRNA processing. The unaffected mother carries only the IVS8-2A>G transition providing evidence that the two changes are in trans-configuration in our patient.\n",
      "Missense mutations in FBN1 exons 41 and 42 cause Weill-Marchesani syndrome with thoracic aortic disease and Marfan syndrome.\tMutations in FBN1 cause a range of overlapping but distinct conditions including Marfan syndrome (MFS), Weill-Marchesani syndrome (WMS), familial thoracic aortic aneurysms/dissections (FTAAD), acromicric dysplasia (AD), and geleophysic dysplasia (GD). Two forms of acromelic dysplasia, AD and GD, characterized by short stature, brachydactyly, reduced joint mobility, and characteristic facies, result from heterozygous missense mutations occurring in exons 41 and 42 of FBN1; missense mutations in these exons have not been reported to cause MFS or other syndromes. Here we report on probands with MFS and WMS who have heterozygous FBN1 missense mutations in exons 41 and 42, respectively. The proband with WMS has ectopia lentis, short stature, thickened pinnae, tight skin, striae atrophicae, reduced extension of the elbows, contractures of the fingers and toes, and brachydactyly and has a missense mutation in exon 42 of FBN1 (c.5242T>C; p.C1748R). He also experienced a previously unreported complication of WMS, an acute thoracic aortic dissection. The second proband displays classic characteristics of MFS, including ectopia lentis, skeletal features, and aortic root dilatation, and has a missense mutation in exon 41 of FBN1 (c.5084G>A; p.C1695Y). These phenotypes provide evidence that missense mutations in exons 41 and 42 of FBN1 lead to MFS and WMS in addition to AD and GD and also suggest that all individuals with pathogenic FBN1 mutations in these exons should be assessed for thoracic aortic disease and ectopia lentis. Further studies are necessary to elucidate the factors responsible for the different phenotypes associated with missense mutations in these exons of FBN1.\n",
      "Bilateral carpal tunnel syndrome in a 9-year-old boy with acromicric dysplasia.\tAcromicric dysplasia is a skeletal dysplasia that is characterized by short stature, short hands and feet, typical facial dysmorphism, normal mental development, and characteristic hand radiology. Carpal tunnel syndrome may be seen in adults with acromicric dysplasia; however, to the authors' knowledge, it has not been reported in pediatric patients. This article describes a 9-year old boy with bilateral carpal tunnel syndrome and acromicric dysplasia treated operatively. No recurrences occurred during 1 year of postoperative follow-up.Carpal tunnel syndrome is a rare disease in childhood. The etiologic factors of carpal tunnel syndrome include trauma (especially distal radius epiphysealis), overuse, genetic or metabolic disorders, space-occupying lesions in the carpal tunnel, hemophilia, congenital anomalies, adverse effect of growth hormone replacement therapy, and idiopathic carpal tunnel syndrome. Acromicric dysplasia should be considered in the etiology of childhood carpal tunnel syndrome.The surgical outcome of carpal tunnel syndrome is good with early diagnosis and treatment. However, in the case of skeletal dysplasia, the diagnosis of carpal tunnel syndrome may be delayed due to anomalies of the hand and due to the child's difficulty in expressing symptoms. Because of the delay in diagnosis of carpal tunnel syndrome in patients with skeletal dysplasia, the treatment outcomes may not be promising. Electrophysiologic studies should be performed early when the clinical signs are positive.\n",
      "From tall to short: the role of TGFÎ² signaling in growth and its disorders.\tThe acromelic dysplasia group is characterized by short stature, short hands and feet, stiff joint, and \"muscular\" build. Four disorders can now be ascribed to this group, namely Weill-Marchesani syndrome (WMS), geleophysic dysplasia (GD), acromicric dysplasia (AD), and Myhre syndrome (MS). Although closely similar, they can be distinguished by subtle clinical features and their pattern inheritance. WMS is characterized by the presence of dislocation of microspherophakia and has autosomal dominant or recessive mode of inheritance. GD is the more severe one, with a progressive cardiac valvular thickening, tracheal stenosis, bronchopulmonary insufficiency, often leading to an early death. AD has an autosomal dominant mode of inheritance, distinct facial and skeleton features (a hoarse voice and internal notch of the femoral head). Finally, MS is sporadic, characterized by prognathism, deafness, developmental delay, thickened calvarium, and large vertebrae with short and large pedicles. We first identified mutations in Fibrillin-1 (FBN1) in the dominant form of WMS and then mutations in A Disintegrin-like And Metalloproteinase domain with ThromboSpondin type 1 repeats 10 (ADAMTS10) in the recessive form of WMS. The function of ADAMTS10 is unknown but these findings support a direct interaction between ADAMTS10 and FBN1. We then identified mutations in ADAMTSL2 in the recessive form of GD and a hotspot of mutations in FBN1 in the dominant form of GD and in AD (exon 41-42, encoding TGFÎ² binding protein-like domain 5 (TB5) of FBN1). The function of ADAMTSL2 is unknown. Using a yeast double hybrid screen, we identified latent transforming growth factor-Î² (TGFÎ²) binding protein 1 as a partner of ADAMTSL2. We found an increased level of active TGFÎ² in the fibroblast medium from patients with FBN1 or ADAMTSL2 mutations and an enhanced phosphorylated SMAD2 level, allowing us to conclude at an enhanced TGFÎ² signaling in GD and AD. Finally, a direct interaction between ADAMTSL2 and FBN1 was demonstrated suggesting a dysregulation of FBN1/ADAMTSL2 interrelationship as the underlying mechanism of the short stature phenotypes. Using exome sequencing in MS probands, we identified de novo SMAD4 missense mutations, all involving isoleucine residue at position 500, in the MH2 domain. In MS fibroblasts, we found decreased ubiquitination level of SMAD4 and increased level of SMAD4 supporting a stabilization of SMAD4 protein. Functional SMAD4 is required for canonical signal transduction through the oligomerization with phosphorylated SMAD2/3 and SMAD1/5/8. We therefore studied the nuclear localization of mutant SMAD complexes and found that the complexes translocate to the nucleus. We finally observed a decreased expression of downstream TGFÎ² target genes supporting impaired TGFÎ² driven transcriptional control in MS. Our findings support a direct link between the short stature phenotypes and the TGFÎ² signaling. However, the finding of enhanced TGFÎ² signaling in Marfan phenotypes supports the existence of yet unknown mechanisms regulating TGFÎ² action.\n",
      "Mutations in the TGFÎ² binding-protein-like domain 5 of FBN1 are responsible for acromicric and geleophysic dysplasias.\tGeleophysic (GD) and acromicric dysplasia (AD) belong to the acromelic dysplasia group and are both characterized by severe short stature, short extremities, and stiff joints. Although AD has an unknown molecular basis, we have previously identified ADAMTSL2 mutations in a subset of GD patients. After exome sequencing in GD and AD cases, we selected fibrillin 1 (FBN1) as a candidate gene, even though mutations in this gene have been described in Marfan syndrome, which is characterized by tall stature and arachnodactyly. We identified 16 heterozygous FBN1 mutations that are all located in exons 41 and 42 and encode TGFÎ²-binding protein-like domain 5 (TB5) of FBN1 in 29 GD and AD cases. Microfibrillar network disorganization and enhanced TGFÎ² signaling were consistent features in GD and AD fibroblasts. Importantly, a direct interaction between ADAMTSL2 and FBN1 was demonstrated, suggesting a disruption of this interaction as the underlying mechanism of GD and AD phenotypes. Although enhanced TGFÎ² signaling caused by FBN1 mutations can trigger either Marfan syndrome or GD and AD, our findings support the fact that TB5 mutations in FBN1 are responsible for short stature phenotypes.\n",
      "Genetic and molecular aspects of acromelic dysplasia.\tThe acromelic dysplasia group includes three rare disorders: Weill-Marchesani syndrome (WMS), Geleophysic dysplasia (GD) and Acromicric dysplasia (AD) all characterized by short stature, short hands and stiff joints. The clinical overlap between the three disorders is striking. Indeed, in addition to the diagnostic criteria, they all share common features including delayed bone age, cone shaped epiphyses, thick skin and heart disease. In contrast, a microspherophakic lens seems to be a characteristic feature of WMS whereas hepatomegaly and a severe outcome are encountered only in the most severe forms of GD. Finally, WMS is transmitted either by an autosomal dominant or an autosomal recessive (AR) mode of inheritance, GD by an autosomal recessive mode of inheritance and AD by an autosomal dominant mode of inheritance. Using genetic approaches, we have identified the molecular basis of WMS and GD which both involved the same superfamily of proteins, the ADAMTS [A Disintegrin-like And Metalloproteinase domain (reprolysin type) with ThromboSpondin type 1 repeats (TSR)]. We have found ADAMTS10 mutations in the recessive form of WMS and Fibrillin 1 mutations in the dominant form of WMS. More recently, we have identified ADAMTSL2 mutations in GD. The function of ADAMTS1 0 and AD AMTSL 2 are unknown. But the findings of FBN1 and ADAMTS10 mutations in WMS suggest a direct link between the two proteins. Using a yeast double hybrid screen, we have identified LTBP1 (Latent TGFbeta Binding protein 1) as a partner of ADAMTSL2. The combination of these findings suggests that ADAMTS10 and ADAMTSL2 are both involved in the microfibrillar network.\n",
      "Subglottic stenosis in short-statured children: a case for further investigation of airway symptoms in patients with skeletal dysplasias.\tClinical evaluation of children with skeletal dysplasias is often concentrated on morphologic and radiographic assessments, but many of these patients also have disease processes of the ear, nose, and throat. We report a case of an 11-month-old girl with an unknown short-limbed dwarfism, similar to acromicric dysplasia, with grade II subglottic stenosis. Laryngotracheoplasty with anterior autologous costal cartilage graft and posterior cricoid split was performed at age 13 months, with subsequent improvement of her airway status. In cases of children with skeletal dysplasias and obstructive airway symptoms, formal otolaryngologic evaluation is warranted for definitive diagnosis and treatment.\n",
      "Acromicric dysplasia: long term outcome and evidence of autosomal dominant inheritance.\tAcromicric dysplasia is a rare bone dysplasia characterised by short stature, short hands and feet, normal intelligence, mild facial dysmorphism, and characteristic x ray abnormalities of the hands. Only a very small number of children with this condition have been reported so far. Here we report on a series of 22 patients including 10 boys and 12 girls with acromicric dysplasia. Length was normal at birth and height fell progressively off the centiles postnatally. The mean adult height was 130 cm (133 cm in males, 129 cm in females). The hands, feet, and limbs were short and OFC was normal. Intelligence was normal and mild dysmorphic features were noted. Other occasional features included well developed muscles, a hoarse voice, generalised joint limitation in some patients, frequent ear, tracheal, and respiratory complication, and spine abnormalities. Long term follow up showed that facial dysmorphism was less obvious in adults and that carpal tunnel syndrome was frequent in older patients. Apart from short metacarpals and phalanges, internal notch of the second metacarpal, external notch of the fifth metacarpal, and internal notch of the femoral heads, there were no major x ray abnormalities. No major complications, such as cardiac disease or major orthopaedic problems, occurred in the course of the disease. The condition appeared to be sporadic in 16 cases but the observation of vertical transmission in three families was consistent with an autosomal dominant mode of inheritance.\n",
      "[Acromicric dysplasia].\n",
      "Acromicric dysplasia and geleophysic dysplasia: similarities and differences.\tWe report on a girl with severe growth retardation, characteristic facies, short stubby hands and feet, progressive joint stiffness, mild aortic and mitral valve insufficiency, and normal intelligence. These features are compatible with the diagnosis acromicric dysplasia. The differential diagnosis with Moore-Federman syndrome and geleophysic dysplasia is discussed; major points to consider in differentiating these entities are the facial appearance, the aspect of the proximal femora, and the presence or absence of storage phenomena. The differences in pattern of inheritance are important in adequate patient care, especially in genetic counselling.\tAcromicric dysplasia, geleophysic dysplasia, and Moore-Federman syndrome may be allelic forms of the same disorder or different disturbances of the same metabolic pathway.\n",
      "[The \"Fantasy Island\" syndrome. Identification of a new osteochondrodysplasia probably of autosomal dominant type].\tClinically, two unrelated patients, an adult male and a female child, coming from non-consanguineous parents, presented dwarfism, peculiar facies, with blepharophimosis, mongoloid slanted eyes, abundant eyebrows and eyelashes, harsh voice and short hands and feet. Radiologically, they presented brachymetacarpalia, brachymetatarsalia, and brachyphalangia of all fingers and toes, shortened and broadened long bones with normal morphology, hypoplastic pelvis and shape anomalies of the vertebral bodies. The clinical and radiological concordance and the differential diagnosis, mainly with acromicric dysplasia, allow the characterization of a distinctive osteochondrodysplasia probably due to an autosomal dominant mutation.\n",
      "Moore-Federman syndrome and acromicric dysplasia: are they the same entity?\tFour unrelated patients are reported with short stature, stiffness of the joints, short fingers, inability to make a fist, and thickened skin on the forearms. Investigations have failed to show a lysosomal storage disorder and radiographs show non-specific changes with a delayed carpal bone age. The clinical features in the four children are very similar to the recently described acromicric dysplasia. There are also similarities to Moore-Federman syndrome which has only been described in one family. The case is made that acromicric dysplasia and Moore-Federman syndrome are the same entity.\n",
      "Geleophysic dysplasia--acromicric dysplasia with evidence of glycoprotein storage.\t\"Geleophysic\" dysplasia is a rare autosomal recessive disorder, probably of glycoprotein metabolism, which shares some clinical and roentgenological manifestations with acromicric dysplasia. We report the clinical, radiological, and pathological data of a patient with the typical picture of progressive growth delay; mild facial anomalies; small, abnormal hands; hepatosplenomegaly; and progressive cardiac valvular lesions. Electron microscopy of a liver biopsy showed similar and additional changes to those published previously.\n",
      "Acromicric dysplasia.\tWe describe a new type of bone dysplasia, the \"acromicric dysplasia,\" based on the study of six patients. This dysplasia is characterized clinically by mild facial anomalies, markedly shortened hands and feet, and growth retardation that is severe in most of cases. Roentgenograms of the hands are characteristic: the metacarpals and the phalanges are short and stubby, the proximal portion of the last four metacarpals are slightly pointed with an external notch on the 2nd metacarpal and an internal notch on the 5th metacarpal, similar to pseudo-epiphysis. The shape of the epiphysis and the metaphysis of the long bones is almost normal, except for a slight deformation of the femoral heads in some patients. No signs of visceral storage were found, which rules out geleophysic dwarfism. The histological, histochemical, and electron microscopical examination of the growth cartilage in two cases showed similar lesions: disorganization of the growth zone with islands of cells, some of them degenerated; abnormal organization of collagen forming thick rims around the cells and wide fibers in the interterritorial matrix; large accumulation of glycogen in most chondrocytes. Both sexes are affected; all patients are isolated cases from normal families.\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'else:\\n    # Print error message\\n    print(\"Error executing command:\")\\n    print(result.stderr)'"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import subprocess\n",
    "import xmltodict\n",
    "import json\n",
    "\n",
    "# https://www.nlm.nih.gov/dataguide/classes/edirect-for-pubmed/samplecode3.html\n",
    "bash_command = 'esearch -db pubmed -query \"ACROMICRIC DYSPLASIA[Title/Abstract]\" | efetch -format xml | xtract -pattern PubmedArticle -element ArticleTitle AbstractText'\n",
    "\n",
    "# | xtract -pattern PubmedArticle -element TitleText AbstractText\n",
    "# | xtract -pattern PubmedArticle -element Abstract\n",
    "\n",
    "# Run the command\n",
    "result = subprocess.run(bash_command, shell=True, capture_output=True, text=True)\n",
    "\n",
    "# Check if the command was successful\n",
    "if result.returncode == 0:\n",
    "    print(result.stdout)\n",
    "    '''articles = result.stdout.split(\"\\n\\n\")\n",
    "    for article in articles:\n",
    "        #print(article)\n",
    "        print(article.split(\"  -\"))\n",
    "        ab_index = article.find(\"A  -\")\n",
    "        print(\"========================\")\n",
    "        #print(article[ab_index:])\n",
    "        break'''\n",
    "        \n",
    "\n",
    "    \n",
    "'''else:\n",
    "    # Print error message\n",
    "    print(\"Error executing command:\")\n",
    "    print(result.stderr)'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5d6b1d94-691a-48e0-a70c-50529de3d432",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
